src/Components/Sidebar/Sidebar.js

```diff
@@ -1,13 +1,48 @@
 import React from 'react';
 import './Sidebar.scss';
 import SidebarProject from '../SidebarProject/SidebarProject';
+import { retrieveProjects } from '../../util/apiCalls';
+// import { useSelector, useDispatch } from 'react-redux';
+// import { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { addProjects } from '../../actions';
 
-export const Sidebar = () => {
-  return (
-    <> 
-      <SidebarProject />
-    </>
-  )
+
+export class Sidebar extends React.Component {
+  componentDidMount() {
+    retrieveProjects()
+      .then(data => {
+        this.props.addProjects(data)
+      })
```

Comment:
Looks like this is what we want to use a hook for in this component.  Check out useEffect Part 1 in the Scrimba course for a walk through on how to refactor this :)
https://scrimba.com/p/p7P5Hd/caqbeZTw